### PR TITLE
Fix py-pox build recipe

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pox/package.py [hunk 1]
```diff
@@ -19,3 +20,13 @@ class PyPox(PythonPackage):
     depends_on('python@2.5:2.8,3.1:')
 
     depends_on('py-setuptools@0.6:', type='build')
+
+    def url_for_version(self, version):
+        url = self.list_url
+        if version >= Version('0.2.5'):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pox/package.py]: This version is not in Spack

Comment (iarspider) [reply var/spack/repos/builtin/packages/py-pox/package.py]: Added

File: var/spack/repos/builtin/packages/py-pox/package.py [hunk 2]
```diff
@@ -10,12 +10,23 @@ class PyPox(PythonPackage):
...
     version('0.2.2', sha256='c0b88e59ef0e4f2fa4839e11bf90d2c32d6ceb5abaf01f0c8138f7558e6f87c1')
     version('0.2.1', sha256='580bf731fee233c58eac0974011b5bf0698efb7337b0a1696d289043b4fcd7f4')
 
     depends_on('python@2.5:2.8,3.1:')
 
     depends_on('py-setuptools@0.6:', type='build')
+
+    def url_for_version(self, version):
+        url = "https://pypi.io/packages/source/p/pox/"
+        if version >= Version('0.2.5'):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pox/package.py]:
```suggestion
        if version >= Version('0.2.4'):
```

File: var/spack/repos/builtin/packages/py-pox/package.py [hunk 3]
```diff
@@ -10,12 +10,23 @@ class PyPox(PythonPackage):
...
+            url += 'pox-{0}.tar.gz'
+        else:
+            url += 'pox-{0}.zip'
+
+        url = url.format(str(version))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pox/package.py]:
```suggestion
        url = url.format(version)
```

**Pull request merged**